src/NativeScript/GlobalObject.moduleLoader.mm

```diff
@@ -64,6 +64,11 @@
         absolutePath = [[absolutePath stringByAppendingPathComponent:path] stringByStandardizingPath];
     }
 
+    WTF::String requestedPath = absolutePath;
+    if (self->modulePathCache().contains(absolutePath)) {
```

Comment:
Use `requestedPath` instead of `absolutePath` here and on the next line - `requestPath` is already converted from an `NSString`.